connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/LiteralExpressionProtoConverter.scala

```diff
@@ -107,7 +107,7 @@ object LiteralValueProtoConverter {
           ArrayType(DataTypeProtoConverter.toCatalystType(lit.getArray.getElementType)))
 
       case _ =>
-        throw InvalidPlanInput(
+        throw new UnsupportedOperationException(
           s"Unsupported Literal Type: ${lit.getLiteralTypeCase.getNumber}" +
             s"(${lit.getLiteralTypeCase.name})")
```

Comment:
I feel that `UnsupportedOperationException` can't describe `Unsupported Literal Type` well.

Reply:
+1, let me revert here